app/views/admin/accounts/show.html.haml

```diff
@@ -78,171 +36,11 @@
-              - else
-                = t 'admin.accounts.security_measures.only_password'
-            %td
-              - if @account.user&.two_factor_enabled?
-                = table_link_to 'unlock', t('admin.accounts.disable_two_factor_authentication'), admin_user_two_factor_authentication_path(@account.user.id), method: :delete if can?(:disable_2fa, @account.user)
-
-          - if can?(:reset_password, @account.user)
-            %tr
-              %td
-                = table_link_to 'key', t('admin.accounts.reset_password'), admin_account_reset_path(@account.id), method: :create, data: { confirm: t('admin.accounts.are_you_sure') }
-
-          %tr
-            %th= t('simple_form.labels.defaults.locale')
-            %td= standard_locale_name(@account.user_locale)
-            %td
-
-          %tr
-            %th= t('admin.accounts.joined')
-            %td
-              %time.formatted{ datetime: @account.created_at.iso8601, title: l(@account.created_at) }= l @account.created_at
-            %td
-
-          - recent_ips = @account.user.ips.order(used_at: :desc).to_a
-
-          - recent_ips.each_with_index do |recent_ip, i|
-            %tr
-              - if i.zero?
-                %th{ rowspan: recent_ips.size }= t('admin.accounts.most_recent_ip')
-              %td= recent_ip.ip
-              %td= table_link_to 'search', t('admin.accounts.search_same_ip'), admin_accounts_path(ip: recent_ip.ip)
-
-          %tr
-            %th= t('admin.accounts.most_recent_activity')
-            %td
-              - if @account.user_current_sign_in_at
-                %time.formatted{ datetime: @account.user_current_sign_in_at.iso8601, title: l(@account.user_current_sign_in_at) }= l @account.user_current_sign_in_at
-            %td
-
-          - if @account.user&.invited?
-            %tr
-              %th= t('admin.accounts.invited_by')
-              %td= admin_account_link_to @account.user.invite.user.account
-              %td
-
+          = render 'admin/accounts/local_account', account: @account
         - else
-          %tr
-            %th= t('admin.accounts.inbox_url')
-            %td
-              = @account.inbox_url
-              = fa_icon DeliveryFailureTracker.available?(@account.inbox_url) ? 'check' : 'times'
-            %td
-              = table_link_to 'search', @domain_block.present? ? t('admin.domain_blocks.view') : t('admin.accounts.view_domain'), admin_instance_path(@account.domain)
-          %tr
-            %th= t('admin.accounts.shared_inbox_url')
-            %td
-              = @account.shared_inbox_url
-              = fa_icon DeliveryFailureTracker.available?(@account.shared_inbox_url) ? 'check' : 'times'
-            %td
-              - if @domain_block.nil?
-                = table_link_to 'ban', t('admin.domain_blocks.add_new'), new_admin_domain_block_path(_domain: @account.domain)
+          = render 'admin/accounts/remote_account', account: @account, domain_block: @domain_block
```

Comment:
Might make sense to convert @domain_block here to a `helper_method` exposed from the controller.